### DISCUSSION TITLE
Are there plan to support TPUs or other hardware?

### DISCUSSION THREAD

**Discussion by ecidon** [microsoft/superbenchmark#568] [Ideas]
Are there plan to support TPUs or other hardware in the benchmarks? what would it take to allow such support? pytorch integration?

**Comment by abuccts** [comment]
Hi @ecidon, currently we only support NVIDIA/AMD GPUs in additional to CPU. For new hardware, you will need the driver, libraries (e.g., CUDA, ROCm), and framework (e.g., PyTorch) supports.
